deploy.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
Let us refactor this file inside `peering.py` so we have only one additional file in the top-level directory of the PEERING client.